src/main/java/seedu/address/model/person/CovidStatus.java

```diff
@@ -37,13 +37,13 @@ public CovidStatus(String status) {
     /**
      * Returns true if a given string is a valid covid status.
      *
-     * @param test
+     * @param inputStatus
```

Comment:
Same as before. :blush:

Reply:
Noted! I've made the appropriate changes 😄